src/main/kotlin/no/nav/familie/tilbake/behandling/domain/Fagsak.kt

```diff
@@ -37,19 +41,19 @@ enum class Ytelsestype(val kode: String) {
 }
 
 enum class Fagsystem(val kode: String, val tema: String) {
```

Comment:
Savner fullt navn etter overgang til kortnavn `(val kode: String, val navn: String val tema: String)` for enklere lesbarhet, og strengt tatt trenger vi vel ikke kode siden denne er lik enum navn.

Reply:
Fagsystem var flyttet til familie-kontrakt og brukte i koden derfra. Gjerne rebase koden når https://github.com/navikt/familie-tilbake/pull/56 er merget til main